### ISSUE TITLE
Threadfin + Jellyfin

### ISSUE THREAD

**Issue by tanmay-pathak** [LegendaryFire/magplex#6]
Hey, thank you for creating this amazing project! 

I am trying to configure magplex with threadfin to get my IPTV working in Jellyfin. I see the filtered channels in the Live TV section of Jellyfin but none of them are playing anything. I just get `Playback Error - Playback failed due to a fatal player error.`. 

Both threadin and magplex logs have nothing regarding this. I am wondering how did you got all of this working? Does your IPTV also check for subscription via the MAC address? 

Thank you!

**Comment by LegendaryFire** [comment]
Seems to work just fine for me with Jellyfin. Sounds like your device is authenticating as well or you would've seen authentication errors. Try pick a random channel from `http://server-ip:5123/channel_list.m3u8`, make sure you have a channel list. From there try get a channel URL with the following endpoint `http://server-ip:5123/channel/<channel_id>`. If that generates fine, try load a network stream in VLC Media Player and see if it plays.

**Comment by tanmay-pathak** [comment]
Yeah, my IPTV service does authenticate via checking if the MAC address is the same. Would magplex support auth like that?

**Comment by LegendaryFire** [comment]
Yep some providers only use the MAC address, other more strict providers will check MAC & DeviceID's however. Are you able to access the channel list file as I mentioned in the last comment?

**Comment by shubhsheth** [comment]
I seem to be running into a similar issue. In the logs I see that magplex is redirecting:
```
"GET /channel/5263 HTTP/1.1" 302 -
```

However the playback doesn't work in Plex or VLC. Not sure if there needs to any additional auth params passed in each request?

**Comment by shubhsheth** [comment]
So there were basically 2 issues. One the channel I was trying didn't work at all. Once I got the right channel it started working on VLC but not Plex.

The second issue was that the stream format wasn't support in Plex. In xteve once I buffered it through FFMEG using this guide it started working: https://www.reddit.com/r/PleX/comments/vkn4tu/comment/ipbnp5l/?utm_source=reddit&utm_medium=web2x&context=3

**Comment by LegendaryFire** [comment]
@shubhsheth It comes down to stream compatibility with Plex regarding FFMPEG buffering unfortunately.

I'm glad you got it working however. I plan on revisiting this project soon to do some more refining, especially the EPG functionality. Keep an eye out over the next few weeks for an update and some improvements! 💯
